app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -1,58 +1,39 @@
 import React from "react";
 
-import { Formik, Form } from "formik";
-import { Button } from "neetoui";
-import { Input, Textarea } from "neetoui/formik";
-import * as yup from "yup";
+import { Input, Select } from "neetoui/v2/formik";
+import { CONTACTS, TAGS } from "./constants";
 
-import notesApi from "apis/notes";
+const NewNoteForm = () => (
+  <div className="w-full space-y-5">
+    <Input
+      label="Title"
+      name="title"
+      placeholder="Enter note title"
+      required={true}
```

Comment:
We don't need to pas `={true}` when we are setting boolean props to true. We can simply provide `required`.